app/src/components/misc/SupportPanel/sendSlackInvite.ts

```diff
@@ -0,0 +1,27 @@
+import { message } from "antd";
+import { getFunctions, httpsCallable } from "firebase/functions";
+
+export default async function sendSlackInvite() {
+  const hide = message.loading({
+    content: "Sending Slack Connect Invitation",
+    className: "slack-connect-toast",
+    duration: 0,
+  });
+  const sendSlackInviation = httpsCallable<null, boolean>(getFunctions(), "slackConnect-sendSlackInvitation");
```

Comment:
https://github.com/requestly/requestly/pull/1968#discussion_r1683979551
is it really required to mention the generic types here - `httpsCallable<null, boolean>` ?